targets/esp8266/source/jerry_run.c

```diff
@@ -45,7 +46,8 @@ int js_entry (const char *source_p, const size_t source_size)
   {
     if ((flags & JERRY_FLAG_PARSE_ONLY) == 0)
     {
-      ret_code = jerry_run ();
+      jerry_api_value_t res;
```

Comment:
It is not mandatory, but good practice to initialize `res` to `VOID` (`jerry_api_create_void_value`).